src/Shared/Corousel/Corousel.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react';
+import img1 from '../../Assets/Images/Blithe1.PNG'
+import img2 from '../../Assets/Images/Blithe2.PNG'
+import img3 from '../../Assets/Images/Blithe-4.PNG'
```

Comment:
suggestion: if you are importing from the same root file like here 'Assets', you can create index.js in the Assets folder, import and export from index.js which is required to use in different places, so the extra lines of code can be removed.